docs/connect-blockchain/near/near.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Integrate Web3Auth with the Near Blockchain
+image: "banners/near.png"
+hide_table_of_contents: true
+description: "Integrate Web3Auth with the Near Blockchain | Documentation - Web3Auth"
+---
+
+import Tiles from "@theme/Tiles";
+
+For Near integration, Web3Auth offers an `ed25519` private key specifically tailored for the Solana
+blockchain. This cryptographic key conforms to the standards required for secure authentication and
+transaction signing on Solana's network.
+
+Integrating Web3Auth with Solana is straightforward and efficient. Developers can leverage
+Web3Auth's functionalities to establish secure connections with Solana nodes, enabling seamless
```

Comment:
Call them Ed25519/ EDDSA nodes rather

Reply:
Have kept nodes, because ed25519 nodes would be confusing, considering not everyone knows about curves.